union-ai-admin/aws/union-ai-admin-role.template.yaml

```diff
@@ -296,7 +300,6 @@ Resources:
               - events:DescribeRule
               - events:ListTargetsByRule
               - events:ListTagsForResource
-              - events:UntagResource
```

Comment:
Is there a reason we are removing these?

Reply:
This has been fixed.